src/NSrtm.Core/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="JetBrains.Annotations" version="9.1.1" targetFramework="net45" developmentDependency="true"/>
+  <package id="JetBrains.Annotations" version="9.1.1" targetFramework="net45" developmentDependency="true" />
```

Comment:
If you do not understand sth plase ask, do not change sth like that (it casues Nuget dependency between NSrtm and JetBrains Annotations).